test/test.py

```diff
@@ -7,14 +7,48 @@
 import os
 
 
+def _extract_digits(s):
+    return int("".join([i for i in s if i.isdigit()]))
+
+
+class AudioBackendScope:
+    def __init__(self, backend):
+        self.new_backend = backend
+        self.previous_backend = torchaudio.get_audio_backend()
+
+    def __enter__(self):
+        torchaudio.set_audio_backend(self.new_backend)
+        return self.new_backend
+
+    def __exit__(self, type, value, traceback):
+        backend = self.previous_backend
+        torchaudio.set_audio_backend(backend)
+
+
 class Test_LoadSave(unittest.TestCase):
+
+    default_audio_backend = torchaudio.get_audio_backend()
+
     test_dirpath, test_dir = common_utils.create_temp_assets_dir()
     test_filepath = os.path.join(test_dirpath, "assets",
                                  "steam-train-whistle-daniel_simon.mp3")
+    test_filepath_wav = os.path.join(test_dirpath, "assets",
```

Comment:
Isn't this the same path as above?

Reply:
`wav` vs `mp3` file, since soundfile doesn't support `mp3`.